### DISCUSSION TITLE
Is it possible to have a local block list file?

### DISCUSSION THREAD

**Discussion by sholland1** [qutebrowser/qutebrowser#6334] [Q&A] [answered]
I created a file with a block rule and added the path to the `content.blocking.adblock.lists` variable like this: `["file:///home/username/test.txt"]`, but qutebrowser doesn't seem to be picking it up. Am I doing something wrong?

**Comment by The-Compiler** [comment, marked as answer]
This [definitely is supported](https://github.com/qutebrowser/qutebrowser/blob/v2.1.1/qutebrowser/components/utils/blockutils.py#L104-L113). It works fine for me, testing with:

```
echo '||qutebrowser.org^' >> /tmp/test.txt && python3 -m qutebrowser --temp-basedir -s content.blocking.adblock.lists '["file:///tmp/test.txt"]' :adblock-update ':later 2000 open qutebrowser.org'
```

What exactly is in your file and what happens after an `:adblock-update`?

**Reply by sholland1** [reply]
Ok. I tried your command, and the images were successfully blocked. I put `||qutebrowser.org^` in my file, and it was successfully read by qutebrowser after running the `:adblock-update` command. My original file contains `qutebrowser.org###_qutebrowser` which should block the first `h2` tag with `id="_qutebrowser"`. Is element blocking not supported yet?

**Reply by The-Compiler** [reply]
It indeed isn't, see #5754
